www/restinio/Portfile

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem                  1.0
+PortGroup                   github 1.0
+PortGroup                   cmake 1.1
+PortGroup                   compiler_blacklist_versions 1.0
+
+github.setup                Stiffstream restinio 0.4.7 v.
+categories                  www devel
+platforms                   darwin
+license                     BSD
+maintainers                 @Lord-Kamina
+description                 Header-only C++14 library that gives you an embedded HTTP/Websocket server.
+depends_build-append        port:boost port:openssl port:pcre port:zlib port:doxygen
+depends_lib-append          port:asio \
+                            port:http-parser \
+                            port:libfmt
+long_description            RESTinio is a header-only C++14 library that gives you an embedded \
+                            HTTP/Websocket server. It is based on standalone version of ASIO and targeted \
+                            primarily for asynchronous processing of HTTP-requests.
+checksums                   rmd160  33f461c5c9c5c7bc5ce9a9daf160875f02c29910 \
+                            sha256  e7500244d5d1f0010d47cbddea524dadf3da64a8c8d7591935f58e5de227e876 \
+                            size    434843
+homepage                    https://stiffstream.com/en/products/restinio.html
+
+patchfiles                  patch-CMakeLists.txt.diff \
+                            patch-Doxyfile.diff
+
+cmake.source_dir            ${worksrcpath}/dev
+cmake.build_type            Release
+
+configure.env-append        ASIO_STANDALONE=1 \
+                            ASIO_HAS_STD_CHRONO=1 \
+                            ASIO_DISABLE_STD_STRING_VIEW=1 \
+                            FMT_HEADER_ONLY=1 \
+                            RAPIDJSON_HAS_STDSTRING=1 \
+                            RAPIDJSON_HAS_CXX11_RVALUE_REFS=1
+configure.pre_args-delete   -DCMAKE_POLICY_DEFAULT_CMP0025=NEW
+configure.optflags-delete   -Os
+cmake.generator             Unix Makefiles
+compiler.blacklist          *gcc-3.* *gcc-4.* {*gcc-5.[0-3]} \
+                            {clang < 800} macports-clang-3.4 macports-clang-3.5 macports-clang-3.6 macports-clang-3.7
```

Comment:
I would be tempted at including the cxx11 1.1 PortGroup and only appending some compilers (we don't have cxx14 PG yet).